### DISCUSSION TITLE
Make "mouse4" go back one page, instead of closing the page

### DISCUSSION THREAD

**Discussion by airone01** [ppy/osu#17911] [Ideas] [answered]
Make "mouse4" (back mouse button) go back one page, instead of closing the whole explorer window.
 The current behavior is that it skips pages when mouse4 is pressed, instead of going back one page at a time.

https://user-images.githubusercontent.com/21955960/164486772-132bc48e-eef7-43ac-893b-365d99b290c9.mp4

In this video you can see that mouse4 closes the wiki and the player page entirely, while always going back to the bm listing if I got through it before.

Additionally we can have a long-press of mouse4 go back to the main menu.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/discussions/13759
